main/lsp/updates.cc

```diff
@@ -293,10 +349,12 @@ LSPLoop::TypecheckRun LSPLoop::tryFastPath(unique_ptr<core::GlobalState> gs,
         ENFORCE(initialGS->errorQueue->isEmpty());
         vector<ast::ParsedFile> updatedIndexed;
         for (auto &f : subset) {
+            unique_ptr<KeyValueStore> kvstore; // nullptr
```

Comment:
LSPLoop has a `kvstore`, but I can't use it here as `indexOne` wants a non-const reference. Thankfully, LSPLoop's `kvstore` is always `null`.

Reply:
can we add an ENFORCE to check the `thankfully` clause ? :-)

Reply:
(If we want to use an actual `kvstore` in future, we could "cheat" by making it an argument to the function.)